governance/default_test.go

```diff
@@ -550,6 +550,67 @@ func TestCypressGenesisHash(t *testing.T) {
 	}
 }
 
+func TestGovernance_initializeCache(t *testing.T) {
+	dbm := database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB})
+	config := getTestConfig()
+	config.Istanbul.Epoch = 30
+
+	gov := NewGovernanceInitialize(config, dbm)
+
+	testData := []struct {
+		// test input
+		governanceUpdateNum int
+		blockNums           []uint64
+		unitPrices          []uint64
+		currentBlockNum     int64
+		// expected result
+		unitPriceInCurrentSet uint64
+		actualGovernanceBlock uint64
+	}{
+		{3, []uint64{30, 60, 90}, []uint64{2, 3, 4}, 90, 3, 60},
+		{3, []uint64{30, 60, 90}, []uint64{2, 3, 4}, 110, 3, 60},
+		{3, []uint64{30, 60, 90}, []uint64{2, 3, 4}, 130, 4, 90},
+	}
+
+	for _, tc := range testData {
+		// 1. initializing
+
+		// store governance items to the governance db for 'tc.governanceUpdateNum' times
+		for idx := 0; idx < tc.governanceUpdateNum; idx++ {
+			config.UnitPrice = tc.unitPrices[idx]
+
+			src := GetGovernanceItemsFromChainConfig(config)
+			delta := NewGovernanceSet()
+
+			if ret := gov.WriteGovernance(tc.blockNums[idx], src, delta); ret != nil {
+				t.Errorf("Error in testing WriteGovernance, %v", ret)
+			}
+		}
+		// store head block (fake block, it only contains block number) to the headerDB
+		header := &types.Header{Number: big.NewInt(tc.currentBlockNum)}
+		gov.db.WriteHeadBlockHash(header.Hash())
+		gov.db.WriteHeader(header)
+
+		// reset idxCache and itemCache. purpose - reproduce the environment of the restarted node
+		gov.idxCache = nil
+		gov.itemCache.Purge()
+
+		// 2. call initializeCache
+		err := gov.initializeCache()
+
+		// 3. check the affected values with expected results
+		assert.Equal(t, nil, err)
+
+		v, ok := gov.currentSet.GetValue(GovernanceKeyMap["governance.unitprice"])
+		assert.Equal(t, true, ok)
```

Comment:
same here. we have assert.True()

Reply:
reflected at https://github.com/klaytn/klaytn/pull/1117/commits/6dfe28836595a514c2c14fb7fead62131f8f74b6.